compiler/rustc_session/src/options.rs

```diff
@@ -1186,12 +1186,13 @@ options! {
 }
 
 options! {
-    DebuggingOptions, DB_OPTIONS, dbopts, "Z", "debugging",
+    UnstableOptions, Z_OPTIONS, dbopts, "Z", "unstable",
```

Comment:
this change wasn't automatic - I think "unstable" is a good new description, and `Z_OPTIONS` isn't super descriptive, but neither was `DB_OPTIONS`.